spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -808,6 +839,54 @@ first20 = products.findByNameLike(name, pageable);
 
 ----
 
+=== Methods With Entity Parameters
+
+Repository methods with a name that begins with one of the prefixes, `insert`, `save`, `update`, `delete`, can have a single entity parameter that is one of the following types, where `E` is the entity type:
+
+- `E` - the entity type
+- `E[]` - an array of the entity type
+- `E...` - a variable arguments array of the entity type
+- `Iterable<E>` and subtypes of `Iterable<E>` - a collection of multiple entities
+
+Note: A form of `delete` and `update` can be defined in a different manner under the Query by Parameters and Query by Method Name patterns. In those cases, the method can have multiple parameters, none of which can be the entity type.
+
+==== Insert Methods
```

Comment:
When discussing a repository, the save is the core method abstraction as a collection.

[Take Oliver's comments](https://github.com/jakartaee/data/issues/229#issuecomment-1701248061)

IMHO: The insert and update methods makes more sense in the DAO pattern.

Reply:
> Take Oliver's comments

The link you posted to Oliver's comments appear to be an argument against allowing a single repository to have methods for a variety of types of entity.  This pull has no relation to that idea.

> IMHO: The insert and update methods makes more sense in the DAO pattern.

I'm documenting how insert and update work because you added insert and update under your pull #260, "determining the query from the supplied parameters and the `insert`, `update`, `delete`, `find`, `count`, or `exists` prefix."  Since we have them in the spec, we need to document how they work.  I just noticed that `save` is absent from that list, which seems like a mistake given that we have save throughout the rest of the spec.  I added https://github.com/jakartaee/data/pull/266/commits/ba3a536e5fe6b870ee05df0eabcdebacd36b5cb4 to this pull to include it.  Please review again.